account/views/client_views.py

```diff
@@ -5,4 +5,4 @@
 def show_profile(request, client_id):
     client = get_object_or_404(Client, pk=client_id)
     profile = client.profile_id
-    return render(request, 'account/profile.html', {'client': client, 'profile': profile})
+    return render(request, 'account/profile_details.html', {'client': client, 'profile': profile})
```

Comment:
This parameter could be retrieved through
 response.context.get("client")
 response.context.get("profile")